### ISSUE TITLE
Create a known issue for thunderbolt device timeout

### ISSUE THREAD

**Issue by hughsie** [hughsie/lvfs-website#68]
For instance, see https://fwupd.org/lvfs/report/275

Maybe something like this:

URI: LVFS-Triaged-Issue:-Thunderbolt-Controller-Timeout
Summary: The Thunderbolt controller did not restart quickly enough
Solution: We think this is harmless. If you upgrade to fwupd from git master we think this is fixed.

**Comment by superm1** [comment]
So I think you should match fwupd version less than 1.0.6 too.

How about:

We think this is harmless and the controller updated properly. This problem causing this error has been fixed in fwupd 1.0.6 and later.

If the controller is still showing the old version after restart please file a bug (link).

**Comment by hughsie** [comment]
Thanks, rules created on the LVFS.

**Comment by superm1** [comment]
I think that this rule should also match this failed report:

```
AppstreamGlibVersion=0.7.5, CpuArchitecture=x86_64, DistroId=fedora, DistroVariant=workstation, DistroVersion=27, FirmwareId=364, Flags=35, FwupdVersion=1.0.5, GUsbVersion=0.2.11, Guid=4eeb9d07-a96c-56d6-92d3-4a23ee7a6e4a, KernelVersion=4.15.3-300.fc27.x86_64, MachineId=fdad171f70dbc0294ae0430df2d05e5d69afc6ad9a2dde6fa7a30e6cc4f0c070, Plugin=thunderbolt, ThunderboltNative=False, UpdateError=could not detect device after update: timed out while waiting for device, UpdateState=failed, VersionNew=28.00, VersionOld=23.00
```

Can you adjust the regex for it?

**Comment by hughsie** [comment]
> FwupdVersion=1.0.5

This is the latest release; i.e. the one with the much longer timeout that waits for the device to come back. That makes it something that we need to investigate further rather than a known issue, at least in my book.

**Comment by superm1** [comment]
OK.

**Comment by superm1** [comment]
The fix for longer timeouts landed after 1.0.5 though too:
https://github.com/hughsie/fwupd/commit/7b19f7db5b7f537171a3193ff3981fb890110095

**Comment by hughsie** [comment]
Issue fixed.
